readme.md

```diff
@@ -1,62 +1,28 @@
----
-topic: sample
-languages:
-- csharp
-products:
-- windows
----
 
-# Windows Community Toolkit
+# Windows Community Toolkit :briefcase:
```

Comment:
We may want to the toolbox emoji 🧰 or our own logo maybe from our [design assets](https://github.com/windows-toolkit/WindowsCommunityToolkit-design-assets)?